src/rp2_common/hardware_uart/include/hardware/uart.h

```diff
@@ -414,7 +405,11 @@ void uart_set_translate_crlf(uart_inst_t *uart, bool translate);
  *  \ingroup hardware_uart
  */
 static inline void uart_default_tx_wait_blocking(void) {
```

Comment:
The fact that this is tied specifically to the `uart_default` (rather than taking a `uart_inst_t` parameter as other functions do) seems a bit limiting, so I did a `git grep` and this function doesn't actually seem to be used anywhere, so could probably be removed?
...oh, I see it's actually used in https://github.com/raspberrypi/pico-examples/blob/master/clocks/hello_resus/hello_resus.c  Does that "make sense" (I know nothing about resus) and does that mean that the hello_resus example won't work if the USB stdio is used instead? :shrug:

Reply:
open a separate PR